lib/rubocop/cop/lint/duplicate_set_enumerables.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Checks for duplicate enumerables in a Set.
+      # This cop detects arguments and use them to check duplacity.
```

Comment:
It'd be good to include in the documentation a list of what type of objects this cop handles. Also duplacity isn't a word :)

Reply:
Sure! I'll change that later, thanks